pytorch_lightning/trainer/data_loading.py

```diff
@@ -217,8 +221,9 @@ def reset_test_dataloader(self, model):
 
         :param model:
         """
-        if not self.is_overriden('test_step'):
-            return
+        # this violate test on `step_end`
```

Comment:
this is needed or you’ll ask for a dataset when there isn’t one and crash

Reply:
reverted